TarponSimulator2017/View/Player.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework.Input;
+
+using Core;
+
+namespace TarponSimulator2017
+{
+	public class Player : Sprite
```

Comment:
Not sure if Player inheriting from Sprite makes sense to me.
More generally, this player class has a strange role.
Nevermind, let's keep it as it, it will be heavily modified in the future.
Just a reminder to rethink a little bit our architecture during the development.